src/main/java/uk/gov/justice/hmpps/prison/api/resource/OffenderResource.java

```diff
@@ -418,6 +419,7 @@ public ResponseEntity<AdjudicationSearchResponse> getAdjudicationsByOffenderNo(@
         @ApiResponse(responseCode = "500", description = "Unrecoverable error occurred whilst processing request.", content = {@Content(mediaType = "application/json", schema = @Schema(implementation = ErrorResponse.class))})})
     @Operation(summary = "Return a specific adjudication")
     @GetMapping("/{offenderNo}/adjudications/{adjudicationNo}")
+    @VerifyOffenderAccess(overrideRoles = {"SYSTEM_USER", "GLOBAL_SEARCH", "VIEW_PRISONER_DATA", "VIEW_ADJUDICATIONS"})
```

Comment:
Moved from `AdjudicationService.findAdjudication` and added `VIEW_ADJUDICATIONS` - this was the only place calling it so seemed ok to move